services/supervisor/src/proto/service.proto

```diff
@@ -56,11 +56,20 @@ message StartService {
 
 // Request for the configuration change
 message ConfigPropose {
+    // The height until which the update configuration procedure should be completed.
     uint64 actual_from = 1;
+    // New configuration proposition.
     repeated exonum.runtime.ConfigChange changes = 2;
 }
 
+// Confirmation vote for the configuration change
 message ConfigVote {
+    // Hash of configuration proposition.
     exonum.Hash propose_hash = 1;
-    bool decision = 2;
+}
+
+// Pending config change proposal and its hash
+message ConfigProposeEntry {
```

Comment:
Do we need it to be a message? It's an internal structure, isn't it?

Reply:
But it stores in the blockchain therefore we have to describe serialization rules for it. 
But such messages may be described outside of service.proto file.